package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "scripts": {
+    "custom-test": "npx ts-node .src/index.ts"
+  },
+  "type": "module"
+}
```

Comment:
Please add `package.json` file using `npm init` command and add `typescript`, `ts-node` packages with `npm i` command.